src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -733,19 +734,17 @@ class PreTrainedModel
     @replace_return_docstrings(output_type=BaseModelOutputWithPoolingAndCrossAttentions, config_class=_CONFIG_FOR_DOC)
     def forward(
         self,
-        input_ids=None,
-        bbox=None,
-        attention_mask=None,
-        token_type_ids=None,
-        position_ids=None,
-        head_mask=None,
-        inputs_embeds=None,
-        encoder_hidden_states=None,
```

Comment:
These arguments `encoder_hidden_states` and `encoder_attention_masks` are not used.

Should I fix tests too or it's out of the scope of this PR ?